src/chttpd_handlers.erl

```diff
@@ -51,6 +54,20 @@ design_handler(HandlerKey, DefaultFun) ->
 %% Internal Function Definitions
 %% ------------------------------------------------------------------
 
+is_disabled(KeyBin) ->
+    BlacklistConfig = config:get("chttpd", "url_handler_blacklist", []),
```

Comment:
How about dedicating whole section in config for this. Than you would be able to choose error message
Something like:
[chttpd_blacklist]
_users = {not_found, "This endpoint is disabled"}
_global_changes = not_implemented